server/src/main/java/io/crate/execution/engine/aggregation/impl/TopKAggregation.java

```diff
@@ -117,7 +133,8 @@ public static void register(Functions.Builder builder) {
     private final DataType<?> argumentType;
 
     private static final int DEFAULT_LIMIT = 8;
-    private static final int MAX_LIMIT = 10_000;
+    private static final int MAX_LIMIT = 5_000;
+    private static final int DEFAULT_CAPACITY = 8192;
```

Comment:
Note: This the max capacity, the data sketch will start with 8 as default and then increase.

Reply:
Why not call it `DEFAULT_MAX_CAPACITY` then?
And probably also refer in the docs as `maxCapacity` if it is not pre-allocated fully

Reply:
👍

Reply:
So should I keep the new default value of 8192 ?

Reply:
> So should I keep the new default value of 8192 ?

Yes

Reply:
👍